### DISCUSSION TITLE
hive catalog 读取表数据报错

### DISCUSSION THREAD

**Discussion by ASanlittleboss** [apache/doris#17092] [A - General / Q&A]
hadoop版本3.1.1
hive版本3.1.3
集群开启了kerberos认证
doris版本1.2.1
场景：
 在使用hive catalog时，可以访问hive库表元数据，但是select表时报错：connect to pdfs failed. error: HdfsRpcException: Failed to invoke RPC call "getFsStats" on server "xxxx:8020" Caused by: HdfsIOException: Cannot initialize client (2): Unknown SASL mechanism;
catalog properties与官网一致（同时提供 HDFS HA 信息和 Kerberos 认证信息，示例如下）
<img width="816" alt="image" src="https://user-images.githubusercontent.com/49724125/221072366-c259c5c8-7c35-4f96-9408-5f127181a50d.png">
也把core-site, hive-site,hdfs-site载入到conf目录下了；
是否有其他的配置项没有添加？

**Comment by LemonLiTree** [comment]
看下fe日志中的具体报错？

**Comment by fsilent** [comment]
请问找到解决方法了吗，我也遇到了相同的报错。
